romana-install/roles/provision/create-aws-hosts/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Generate CloudFormation Template
+  template: src="aws_cf_small_cluster.json" dest="{{ stack_data_dir }}/cloudformation.json"
```

Comment:
It would be great if those auto-generated files would be able to go into a "_tmp" (or otherwise obvious) directory, so that someone looking at the files in the directory could never be confused about where those files came from. And I think that's possible for all files, even the Vagrantfile: Supposedly you can use the VAGRANT_VAGRANTFILE environment variable to set a custom Vagrantfile location: http://stackoverflow.com/a/29255917